docs/TuxxAnimations.md

```diff
@@ -68,12 +72,15 @@ The only other step to implementing a Tuxx animation is to wrap the element you
 ```javascript
   render: function () {
     return (
-      <Fly>
+      <Fade>
         <h1>Hello World</h1>
-      </Fly>
+      </Fade>
     );
   }
 ```
+
+![Animation 1](http://cdn.makeagif.com/media/2-07-2015/LxdwaO.gif)
+
 ##### <a id="Specifying-an-animation-key"></a>Specifying an Animation Key via the Id Property - type: STRING, ARRAY of STRINGs, or NUMBER - optional [#](#Specifying-an-animation-key)
 Tuxx Animations work by concatenating the children of the animation into an array and wrapping that in a ReactTransitionGroup.  Because of that it is beneficial to provide the animation with an id property, which will tell it where to look in its children's `props` for a key to use.  If no id is provided the animations will still work as long as the children are not an array, but you will see warnings from React regarding the lack of a key.  You can specify a string or array of strings for the id property:
```

Comment:
Change the number of spaces after periods to one in this section and all others in this document..